src/Future.js

```diff
@@ -35,12 +35,12 @@ Future.prototype.ap = function(m) {
       }
     }
 
-    self.fork(doReject, function(fn) {
+    self._fork(doReject, function(fn) {
       applyFn = fn;
       resolveIfDone();
     });
 
-    m.fork(doReject, function(v) {
+    m._fork(doReject, function(v) {
```

Comment:
I'm not too concerned about interop between other "forkables", as to the best of my knowledge that has never been a goal of `Future`. The type passed to `ap` should also be of the same type as the instance whose method is being called, so this isn't an issue for me (but thanks for raising).